src/calculateRentalCost.js

```diff
@@ -4,7 +4,23 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dailyPay = 40;
+
+  if (days <= 2) {
+    return days * dailyPay;
+  }
+
+  let cost = dailyPay * days;
+
+  if (days >= 3 && days <= 6) {
+    cost = cost - 20;
```

Comment:
The discount logic for days between 3 and 6 is correct, but you could simplify the condition by checking `else if (days <= 6)` since the previous condition already checks for `days <= 2`. This would make the code slightly more efficient.